packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss

```diff
@@ -63,30 +71,32 @@ slot[name='left-side'] > .shifted-up,
     flex-basis: 100%;
     display: flex;
     align-items: baseline;
-    margin-bottom: 6px;
-    min-width: 170px;
+    min-height: var(--spacing-10); //40px
+    padding-bottom: var(--spacing-1);
+    min-width: 10.625rem;
 }
 
 .app-meta h1 {
-    margin-bottom: 0;
-    margin-top: 0;
-    height: 32px;
+    margin-bottom: var(--spacing-0);
+    margin-top: var(--spacing-0);
     font-family: var(--font-heading-1-font-family);
     font-size: var(--font-heading-1-font-size);
     font-weight: var(--font-heading-1-font-weight);
     letter-spacing: var(--font-heading-1-letter-spacing);
+    line-height: var(--line-height-2xl);
```

Comment:
is there a specific reason for --line-height-2xl over --font-heading-1-line-height?

Reply:
Nope, just an oversight. Good catch!